.travis.yml

```diff
@@ -5,15 +5,22 @@ python:
   - "pypy"
 before_install:
   - "pushd .."
-  - "git clone https://github.com/pulp/pulp.git --branch 2.5-release"
+  - "echo 'Remove this'"
```

Comment:
On 02/20/2015 05:44 PM, Austin wrote:
> Was this a note to yourself?

Yeah, the lines below this are just temporary so that I can test the
changes I made in platform. I don't want this or the lines below it to
actually be merged, but I wanted to make sure what I was doing in that
other PR made this one pass in Travis. Once that PR is merged, I will
remove these lines before I merge. The PR description has something
along those lines in it as well.